tracing-forest/src/layer/mod.rs

```diff
@@ -50,6 +52,12 @@ impl OpenedSpan {
                 return;
             }
 
+            #[cfg(feature = "defer")]
+            if field.name() == "defer" {
+                defer = true;
```

Comment:
nit: it feels odd to me that you can write `info_span!("something", debug = anything)`, but because we only have a `dyn Debug` it's difficult to do otherwise I guess